CHANGELOG.md

```diff
@@ -11,8 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ### Changed
-
-- Changed all file line endings from CRLF to LF and added the .gitattributes to enforce it - [#163](https://github.com/DigitalExcellence/dex-backend/issues/163)
+- Changed footer.component.html and footer.component.scss for creating a responsive footer. - [#163] (https://github.com/DigitalExcellence/dex-frontend/issues/163) 
```

Comment:
You can just refer to the footer, no need to specify exactly which files you edited. I would phrase this like: `Changed footer styling to be responsive on small screens`